chat-widget/src/components/livechatwidget/common/startChat.ts

```diff
@@ -134,9 +135,6 @@ const initStartChat = async (chatSDK: any, dispatch: Dispatch<ILiveChatWidgetAct
                 }
             });
             isStartChatSuccessful = false;
-            // Resetting the widget state to Closed, for recent introduction of OC rate limiting(429 Error) 
```

Comment:
Why are we removing this?

Reply:
To keep on loading pane on exception, not automatically show chat button